File: RolePlayGame/dotnet/fixtures/my.examples/CalculatorThreeFixture.cs
using fitlibrary;

namespace my.examples {

	public class CalculatorThreeFixture : DoFixture {
	
		public double QuotientOfAndIs(double numerator, double denominator){
			Calculator calculator = new Calculator(); // system under test
			return calculator.QuotientFor(numerator, denominator);
		}
	}
}

File: RolePlayGame/java/fixtures/my/examples/CalculatorThreeFixture.java
package my.examples;

import fitlibrary.DoFixture;

public class CalculatorThreeFixture extends DoFixture {

	public double quotientOfAndIs(double numerator, double denominator){
		Calculator calculator = new Calculator(); // system under test
		return calculator.quotientFor(numerator, denominator);
	}
}
